mono-traversable/src/Data/MonoTraversable.hs

```diff
@@ -1382,3 +1384,25 @@ ointercalate x = mconcat . List.intersperse x . otoList
         ointercalate x = T.intercalate x . otoList #-}
 {-# RULES "intercalate LText" forall x.
         ointercalate x = TL.intercalate x . otoList #-}
+
+-- | Provides a `MonoFoldable`, `MonoFunctor` or `MonoPointed` for an arbitrary
+-- `F.Foldable`, `Functor` or `Applicative`.
+--
+-- Useful for, e.g., passing a `F.Foldable` type you don't own into a
+-- function that expects a `MonoFoldable`.
+--
+-- > // package A
+-- > data MyList a = MyList [a] deriving Foldable
+-- >
+-- > // package B
+-- > process :: MonoFoldable mono => mono -> IO ()
+-- >
+-- > // package C
+-- > process (WrappedMono (MyList []))
```

Comment:
Please also add a minor version bump in the cabal file, a ChangeLog entry, and a `@since` Haddock comment here.

Reply:
Done